name-service/js/.eslintignore

```diff
@@ -0,0 +1,5 @@
+docs
+lib
+test-ledger
+
+package-lock.json
```

Comment:
there's no package-lock.json, so maybe we don't need this line?

Reply:
We use this in token-js as well, for anyone who might do `npm i`. I figured best to follow pattern?

Reply:
ah gotcha, that's fine then, let's leave it in